samples/snippets/create_phrase_matcher_all_of.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# [START contactcenterinsights_create_phrase_matcher_all_of]
+from google.cloud import contact_center_insights_v1
+
+
+def create_phrase_matcher_all_of(
+    project_id: str,
+) -> contact_center_insights_v1.PhraseMatcher:
+    # Construct a parent resource.
+    parent = contact_center_insights_v1.ContactCenterInsightsClient.common_location_path(
+        project_id, "us-central1"
+    )
+
+    # Construct a phrase matcher that matches all of its rule groups.
+    phrase_matcher = contact_center_insights_v1.PhraseMatcher()
+    phrase_matcher.display_name = "NON_SHIPPING_PHONE_SERVICE"
+    phrase_matcher.type_ = (
+        contact_center_insights_v1.PhraseMatcher.PhraseMatcherType.ALL_OF
+    )
+    phrase_matcher.active = True
+
+    # Construct a rule group to match the word "PHONE" or "CELLPHONE", ignoring case sensitivity.
+    rule_group1 = contact_center_insights_v1.PhraseMatchRuleGroup()
```

Comment:
Nit: is there a more descriptive name that could be used for these rule groups?

Reply:
done! I updated the var names to be more descriptive.